code/world.dm

```diff
@@ -108,8 +112,9 @@
 				n++
 		return n
 
-	else if("ircstatus" in input)
-		if(world.time - last_irc_status < IRC_STATUS_THROTTLE)
+	else if("ircstatus" in input)	//tgs2 support
+		var/static/last_irc_status = 0
+		if(world.time - last_irc_status < 50)
```

Comment:
It's gonna get removed eventually, once the new tools are 100% reliable. I wanted to move the define to the __DEFINES tree